docs/reference/localization.md

```diff
@@ -2,193 +2,256 @@
+### A few important notes about localization with Fluent/React
 
-* During a Docker image build, the [fxa-content-server-l10n repo is cloned][#docker-l10n-clone] and the build process started.
-* The build process starts the [l10n-create-json grunt task][#l10n-create-json-grunt-task].
-* The l10n-create-json task copies all of the translated .po files from the fxa-content-server-l10n repo and runs the [grunt po2json task][#grunt-po2json-task], once for each of the 2 .po files for each locale, resulting in 2 .json files for each locale.
-* A copy of the compiled JavaScript bundles are created for each locale.
-  * For each locale, client.json is injected into the [special comment in app/lib/translator.js][#translator-special-comment].
-  * SRI hashes for any dynamically loaded scripts are [added to the JS][#add-sri-to-js].
-* A copy of the .html templates are created for each locale in [server/templates/pages/dist][#dist-template-dir].
-  * The SRI hash of that locale’s JavaScript bundle is [added to the script tag in the HTML][#add-sri-to-html].
+#### Attributes
+
+Attributes can be localized declaratively with `<Localized>` or `<FtlMsg>` components. See the [Fluent Wiki for examples and notes](https://github.com/projectfluent/fluent.js/wiki/Localized#attributes) on the importance of including the `attrs` prop. If `attrs` is not passed, no attributes will be set!
+
+An alternative is to use a hook (either `useLocalization` or `useFtlMsgResolver`) to pass the localized string as a prop. In this scenario, the FTL id should include the name of the attribute (e.g., component-name-image-alt-text) to provide context for localizers.
+
+#### Markup/embedded elements in strings
+
+If the string contains markup (for example, a link), the `elems` prop MUST be used. See [Fluent Wiki for examples and notes](https://github.com/projectfluent/fluent.js/wiki/Localized#markup).
+
+#### Using styling to uppercase/lowercase localized text
+
+Just don't - locales may have different rules around capitalization. Best to apply capitalization to the default 'en' string with a comment about the context, and let localizers determine what is best for their locale.
+
+See [notes about paramtrization for brand terms](https://mozilla-l10n.github.io/localizer-documentation/tools/fluent/basic_syntax.html#terms-and-parameterized-terms).
+
+### Documentation about using Fluent:
```

Comment:
Could be worth adding the Bedrock link too: https://bedrock.readthedocs.io/en/latest/l10n.html#fluent